osmosis-installer.py

```diff
@@ -204,7 +254,13 @@ def startReplayNow():
 1) Yes, start cosmovisor as a background service and begin replay
 2) No, exit and start on my own (will still auto update at upgrade heights)
     """+ bcolors.ENDC)
-    startNow = input(bcolors.OKGREEN + 'Enter Choice: '+ bcolors.ENDC)
+    if args.sr == True :

```

Comment:
Please choose more descriptive names instead of abbreviations

Reply:
Great call, I have no idea why I didn't do this from the start. Made it super difficult to read but now its much better, thanks!